gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,6 @@
 #Thu Dec 02 03:48:33 CET 2021
 distributionBase=GRADLE_USER_HOME
 distributionUrl=https\://services.gradle.org/distributions/gradle-7.5.1-bin.zip
-distributionSha256Sum=f6b8596b10cce501591e92f229816aa4046424f3b24d771751b06779d58c8ec4
```

Comment:
android studio tells you it might make issues to verify the hashsum, but there is one option (the second one) to keep the sha, and it works just fine in android studio then (at least here)

Reply:
Android studio would not build unless I removed this.